### ISSUE TITLE
Support multiple users

### ISSUE THREAD

**Issue by TheColin21** [jellyfin/jellyfin-webos#168]
It would be great for the app to have a "profile" screen similar to Netflix/D+/Prime Video.
For a setup which multiple users use, separate viewing progresses are much needed.

**Comment by BrandonG777** [comment]
other platforms have this feature, not sure why it's being overlooked here but been waiting on this for awhile to fully move over to Jellyfin.

**Comment by anthonylavado** [comment]
It's not being overlooked, but it's actually a much more complicated feature that requires many changes in jellyfin-web, and then some on the server side too.

**Comment by distante** [comment]
> It's not being overlooked, but it's actually a much more complicated feature that requires many changes in jellyfin-web, and then some on the server side too.

What are those changes? I was trying to do some research yesterday here https://github.com/jellyfin/jellyfin-web/issues/5074

But I haven't found any documentation on how the data flow occurs.

**Comment by Deagarys** [comment]
I switched from the android tv client to the webos client, and i really miss the switch profile option. Only way to do this now is setting the profiles to show up on log in page...

Would love to see a profile switcher of logged in users here aswell.

**Comment by Matesaktesak** [comment]
I'd like to bump this feature request as it's very important to the general usability of the app. Not being able to easily switch profiles is the reason why most of my family don't want to use Jellyfin.

**Comment by asbjorjo** [comment]
> > I switched from the android tv client to the webos client, and i really miss the switch profile option. 
> 
> Does that mean that this option exists on Android TV?

Yes, I've been using it with the app on my TV provider's set top box (runs Android) for a couple of years.

**Comment by BrandonG777** [comment]
Switch profile is also available on Emby for all clients including WebOS. I’d love to switch over to Jellyfin but this is a deal breaker for me. Sadly at this point I think my TV will hit EOL before this is addressed.

**Comment by davidmurdoch** [comment]
I hacked together a version of Jellyfin Web with a working logged in user switcher in about an hour.

After a user logs in it stores their information and accessToken in localStorage. I believe this is how the Android TV version does it (obviously not in localStorage, but in some other local db).

I still need to handle log out, and add a settings toggle to enable/disable the feature, for this to be worth sharing, which I'll hopefully be able to do tomorrow. 

Any idea what the requirements are for getting something like this merged?

**Comment by davidmurdoch** [comment]
@anthonylavado, can you chime in on the feasibility of the way I've implemented the proof of concept PR?

**Comment by RasmusFangel** [comment]
A Google search lead me here. Would love to see this feature as well. Anything I can do to help?

**Comment by RasmusFangel** [comment]
Yeah, absolutely - I'm relatively new to jf, but I've got your PoC which is going to save me a lot of time. It looks like you have been waiting a looong time for a list of requirements so I may just give it a go and attempt to get it reviewed. Good work @davidmurdoch !

**Comment by davidmurdoch** [comment]
@RasmusFangel feel free to refine and polish it and submit your own PR. I just didn't want to do a bunch of work to have to change it all if it wasn't up to the Jellyfin maintainer's expectations/requirements.

**Comment by jscottelblein** [comment]
> Good evening, I'm very interested in this topic, any news about it? Thank you

The repo owners/managers completely blew it off.

**Comment by JordanMajd** [comment]
This is an essential feature blocking me from switching to jellyfin. I'm not going to log in and out of my TV every time my kid wants to watch a show

**Comment by BrandonG777** [comment]
> > Good evening, I'm very interested in this topic, any news about it? Thank you
> 
> The repo owners/managers completely blew it off.

Two years later since I posted my original request. I bought another Apple TV 4K thinking it would solve the problem. Jellyfin lacks an official client on that platform. There is a 3rd party app but I wasnt impressed. Emby has fully developed official clients that have the features needed to be useful. Jellyfin has come a long ways but until it has some decent client software, I've decided to just stick with Emby. I did pay for lifetime subscription when I moved away from Plex, not sure if you need that or not but worth it IMO for good clients. It does baffle me how popular Jellyfin is so popular with such few client options.

**Comment by DarthJahus** [comment]
Yes please!
